Makefile

```diff
@@ -3,86 +3,51 @@ export PATH := $(PATH):$(GOPATH)/bin
 INTEGRATION     := nginx
 BINARY_NAME      = nri-$(INTEGRATION)
 SRC_DIR          = ./src/
-VALIDATE_DEPS    = golang.org/x/lint/golint
-TEST_DEPS        = github.com/axw/gocov/gocov github.com/AlekSi/gocov-xml
 INTEGRATIONS_DIR = /var/db/newrelic-infra/newrelic-integrations/
 CONFIG_DIR       = /etc/newrelic-infra/integrations.d
 GO_FILES        := ./src/
+GOFLAGS			 = -mod=vendor
+GOLANGCI_LINT	 = github.com/golangci/golangci-lint/cmd/golangci-lint
+GOCOV            = github.com/axw/gocov/gocov
+GOCOV_XML		 = github.com/AlekSi/gocov-xml
 
 all: build
 
 build: clean validate compile test
 
 clean:
-	@echo "=== $(INTEGRATION) === [ clean ]: removing binaries and coverage file..."
-	@rm -rfv bin coverage.xml
+	@echo "=== $(INTEGRATION) === [ clean ]: cleaning workspace..."
+	@rm -rfv bin coverage.xml vendor
 
-validate-deps:
-	@echo "=== $(INTEGRATION) === [ validate-deps ]: installing validation dependencies..."
-	@go get -v $(VALIDATE_DEPS)
+vendor/modules.txt:
+	@echo "=== $(INTEGRATION) === [ dependencies ]: downloading vendor dependencies..."
+	@go mod vendor
```

Comment:
`go build` should already take care of this. Why do we have a target to generate a vendor folder?

Reply:
I was trying to keep it explicit, but you're right.

Reply:
I'm not sure if the command `go build` will download the dependencies in the  folder `tools`...   The go build command seems to check only  files (*.go)  in the src folder.